src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -63,7 +58,7 @@ public function saveOrder(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer
      *
      * @return void
      */
-    public function saveOrderPayment(QuoteTransfer $quoteTransfer, SaveOrderTransfer $saveOrderTransfer): void;
+    public function saveOrderPayment(QuoteTransfer $quoteTransfer, SaveOrderTransfer $saveOrderTransfer);
```

Comment:
Revert all changes to the return types, the Facade is public API and can't be changed outside of a major release for that reason. Why did you remove them?

Reply:
Reverted. I was a mistake.